CONTRIBUTING.md

```diff
@@ -32,6 +32,17 @@ pull request is most likely to be accepted if it:
 * Has a [good commit
   message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
 
+## Cutting a Release
+
+* Send a pull request with the commit comment: "Release X.Y.Z" against dev including:
+  * update CHANGELOG.md
+  * update JaegerClientVersion in constants.go
+* Send a pull request for dev into master
+* Create new tag "vX.Y.Z" and copy CHANGELOG.md fragment into release notes on
+  https://github.com/uber/jaeger-client-go/releases
+* Send a pull request with the commit comment: "Back to development" against dev including:
+  * update CHANGELOG.md
```

Comment:
I thought:
  1. Create PR "Release X.Y.Z" against `master` branch
     * update change log (should we clarify what to update?, e.g. replace Unreleased with the date)
     * update version in constants
  1. Create a new release on Github (which auto-creates the tag)
  1. Create "Back to development" commit
     * add next release ver to changelog: x.y.z (Unreleased)
     * bump version in constants

It's these changes to the changelog and constants file that are tedious and easy to mess up. zest release was very useful to manage them.